src/components/CarouselImage/index.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { uniqueId } from '../../libs/utils';
+import RenderIf from '../RenderIf';
+import { Consumer } from '../CarouselCard/context';
+import './styles.css';
+
+class Item extends Component {
+    constructor(props) {
+        super(props);
+        this.carouselImageID = uniqueId('carousel-content-id');
+        this.carouselIndicatorID = uniqueId('indicator-id');
+        this.itemRef = React.createRef();
+    }
+
+    componentDidMount() {
+        const { privateRegisterChild, header } = this.props;
+        return setTimeout(() => (
+            privateRegisterChild({
+                containerID: this.carouselImageID,
+                indicatorID: this.carouselIndicatorID,
+                ref: this.itemRef,
+                header,
+            })), 0);
+    }
+
+    getContainerClassName() {
+        const { className } = this.props;
+        return classnames('rainbow-carousel-image_container', className);
+    }
+
+    getTabIndex() {
+        const { activeItem } = this.props;
+        if (activeItem === this.carouselIndicatorID) {
+            return 0;
+        }
+        return -1;
+    }
+
+    render() {
+        const { assistiveText, description, header, href, src, activeItem } = this.props;
+        return (
+            <div
+                id={this.carouselImageID}
+                className={this.getContainerClassName()}
+                role="tabpanel"
+                aria-hidden={activeItem !== this.carouselIndicatorID}
+                aria-labelledby={this.carouselIndicatorID}>
+                <a
+                    href={href}
+                    className="rainbow-carousel-image"
+                    tabIndex={this.getTabIndex()}
+                    ref={this.itemRef}>
+                    <div>
+                        <img className="rainbow-carousel-image_image" src={src} alt={assistiveText} />
+                    </div>
+                    <RenderIf isTrue={header || description}>
```

Comment:
same here create a const hasContent and remember to force this value to be boolean because here the RenderIf will print a warning in the console since always what get passed as isTrue is header or description and these are not booleans.